ceph/ceph_admin/iscsi.py

```diff
@@ -42,11 +47,11 @@ def apply(self, config: Dict) -> None:
                     unmanaged: true
 
         """
-        pos_args = config.pop("pos_args")
-
-        if isinstance(pos_args[-1], list):
-            trusted_list = repr(" ".join(pos_args.pop()))
-            pos_args.append(trusted_list)
-
-        config["pos_args"] = pos_args
+        args = config.get("args")
+        trusted_ip_list = args.get("trusted_ip_list")
```

Comment:
please add a comment that `trusted_ip_list` is no more a positional arg in 5.1.

Reply:
Done. I verified its not a positional arg in 5.x latest releases 50Z2 and 5.1

Reply:
looks like trusted IP list is supported as pos_arg and keyword arg.
``` 
In 5.1 
[ceph: root@magna111 /]# ceph orch apply iscsi iscsi api_user api_passwd "10.8.128.110,10.8.128.111" --placement="magna116,magna126"
Scheduled iscsi.iscsi update...
[ceph: root@magna111 /]# ceph orch ls iscsi
NAME         PORTS  RUNNING  REFRESHED  AGE  PLACEMENT          
iscsi.iscsi             2/2  7s ago     18s  magna116;magna126  
[ceph: root@magna111 /]# ceph version 
ceph version 16.2.7-6.el8cp (5a5143a7ab533ea00cfd5a5f47cf531223dd5b43) pacific (stable)
```

Reply:
Yes, this can be used as Keyword arg or positional arg. We have implemented using keyword arg for this PR.